src/index.html

```diff
@@ -11,12 +11,102 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,400&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
```

Comment:
<img width="825" alt="Знімок екрана 2024-08-13 о 17 02 44" src="https://github.com/user-attachments/assets/ecbf75bf-4b93-48a4-b64b-faad59c70498">

For example:
```suggestion
          alt="MOYO Logo Header"
```

Reply:
> <img alt="Знімок екрана 2024-08-13 о 17 02 44" width="825" src="https://private-user-images.githubusercontent.com/109915550/357440499-ecbf75bf-4b93-48a4-b64b-faad59c70498.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MjEzMDEsIm5iZiI6MTcyMzYyMTAwMSwicGF0aCI6Ii8xMDk5MTU1NTAvMzU3NDQwNDk5LWVjYmY3NWJmLTRiOTMtNDhhNC1iNjRiLWZhYWQ1OWM3MDQ5OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxNFQwNzM2NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01Yzg3NTc2MzUxNjlkYmU3MmE0NzRkNmY1NjJmNDM2Njg3MjMzOTc5YzE3YTU4ZWFiOTg3MDZlYTU3MzVlYmQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nLnh_7MEM6nVoMduq1uuBT4jQzrwLCLLGxpViBSWZ70">
> For example:

Not fixed without any comment